front/app/components/CityLogoSection/index.tsx

```diff
@@ -75,7 +75,7 @@ const CityLogoSection = ({
                 {tenantImage}
               </LogoLink>
             ) : (
-              { tenantImage }
+              tenantImage
             )}
```

Comment:
@IvaKop Just curious what this fix is for?

Reply:
There was a problem unrelated to this PR that was introduced to master where the logo was crashing..

Reply:
Ahh ok :+1: is it alright to have this included in this PR?

Reply:
I merged master and it's not anymore. But good that you spotted it 👍